source/brailleDisplayDrivers/hims.py

```diff
@@ -190,12 +190,13 @@ def registerAutomaticDetection(cls, driverRegistrar: bdDetect.DriverRegistrar):
 		# Bulk devices
 		driverRegistrar.addUsbDevices(bdDetect.DeviceType.CUSTOM, {
 			"VID_045E&PID_930A",  # Braille Sense & Smart Beetle
-			"VID_045E&PID_930B",  # Braille EDGE 40
+   			"VID_045E&PID_930B",  # Braille EDGE 40
 		})
 
 		# Sync Braille, serial device
 		driverRegistrar.addUsbDevices(bdDetect.DeviceType.SERIAL, {
 			"VID_0403&PID_6001",
+   			"VID_1A86&PID_55D3",  # Braille Emotion 40

```

Comment:
Please remove the spaces here as well
```suggestion
			"VID_1A86&PID_55D3",  # Braille Emotion 40
```